plugins/flutter_aepconsent/README.md

```diff
@@ -0,0 +1,50 @@
+# flutter_aepconsent
+
+[![pub package](https://img.shields.io/pub/v/flutter_aepassurance.svg)](https://pub.dartlang.org/packages/flutter_aepconsent) ![Build](https://github.com/adobe/aepsdk_flutter/workflows/Dart%20Unit%20Tests%20+%20Android%20Build%20+%20iOS%20Build/badge.svg) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+`flutter_aepconsent` is a flutter plugin for the iOS and Android [AEPConsent SDK](https://aep-sdks.gitbook.io/docs/foundation-extensions/consent-for-edge-network) to allow for integration with Flutter applications. Functionality to enable the Assurance extension is provided entirely through Dart documented below.
```

Comment:
Please update all occurrences of assurance in this file

Reply:
Updated Readme.